CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## [3.2.0](https://github.com/theforeman/puppet-pulpcore/tree/3.2.0) (2021-03-22)
```

Comment:
Slight nit: this is either outdated or the tag should be backdated. I'd prefer to update it here:
```suggestion
## [3.2.0](https://github.com/theforeman/puppet-pulpcore/tree/3.2.0) (2021-03-23)
```